src/c++/examples/CMakeLists.txt

```diff
@@ -271,7 +271,7 @@ if(TRITON_ENABLE_CC_GRPC)
       simple_grpc_cudashm_client
       PRIVATE
         grpcclient_static
-        ${CUDA_LIBRARIES}
+        CUDA::cudart
```

Comment:
This is the real fix--the `${CUDA_LIBRARIES}` variable was empty, because it's an older variable from when `findCUDA()` was being used, but we use `find_package(CUDAToolkit)` now, which doesn't populate a `${CUDA_LIBRARIES}` variable, it makes a `CUDA::cudart` target:

https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html#cuda-runtime-library